ldk/javascript/examples/self-test-loop/src/tests/whisper/autocomplete/index.tsx

```diff
@@ -73,6 +73,7 @@ export const testAutocomplete = (): Promise<boolean> =>
             multiple: true,
             onSelect: (error, value, onSelectWhisper) => {
               console.log(`Received selected value: ${JSON.stringify(value)}`);
+              console.log('type of value: ', typeof value);
```

Comment:
Is this necessary for testing? The value should always be the same type

Reply:
@JosefBud-olive  Well I believe this is to test a GH issue, and I forgot to delete it 0.0.